CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# dbt_unified_rag v0.1.0-a7
+[PR #25](https://github.com/fivetran/dbt_unified_rag/pull/25) introduces the following updates that **require a full refresh**.
```

Comment:
Since there are multiple PRs in this release, should we change the formatting.

```suggestion
This release introduces the following updates that **require a full refresh**.
```

Then add the PR link to the end of every sentence.